terraform/modules/daos_server/templates/daos_startup_script.tftpl

```diff
@@ -65,40 +65,21 @@ if [[ "$${HOSTNAME}" == "$${FIRST_DAOS_SERVER_HOSTNAME}" ]]; then
   echo "Formatting storage on servers: ${servers}"
   dmg storage format
   dmg system query -v
-  
-
-  while true
-  do
-    if [[ $(dmg system query -v | grep Joined | wc -l) -eq ${instances} ]]; then
-      printf "\n%s\n" "DAOS storage format finished"
-      dmg system query -v
-      break
-    fi
-    printf "%s" "."
-    sleep 5
-  done
-  
```

Comment:
Please add the wait for format logic back in.

Reply:
I noticed that in current DAOS 2.0.2 release, command dmg system query -v is waiting for the servers to be in "Joined" state. So there is no need to add special wait functionality like this. @johannlombardi could you confirm this?

Reply:
@lsitkiew  per the chat we are ok to remove it.